src/sonic-config-engine/minigraph.py

```diff
@@ -1775,9 +1775,7 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
         # update the port lanes, use the first 4 lanes of the 400G port to support 100G/40G port
         if port_default_speed == '400000' and (port_png_speed == '100000' or port_png_speed == '40000'):
             port_lanes =  ports[port_name].get('lanes', '').split(',')
-            # check if the 400g port has only 8 lanes
-            if len(port_lanes) != 8:
-                continue
+
             updated_lanes = ",".join(port_lanes[:4])
```

Comment:
Would it better to update lane map when the number of lanes is 8? Is taking first 4 lanes always right?

Reply:
in both cases when a port has 4 lanes or 8 lanes, if we want to transition from 400G to 100G, we need only first 4 lanes, so we are right in that regard.